### PR TITLE
Header

### PR DESCRIPTION
**Pull request by vivientam**
i hv tried to add underline on header .... but the contact page has problem.

### CODE REVIEW

File: app/helpers/contacts_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module ContactsHelper
```
Comment (h8rry) [inline app/helpers/contacts_helper.rb]: remove this file

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -32,22 +32,18 @@
       </div>
```
Comment (harrychenca) [inline app/views/layouts/_header.html.erb]: Some indentations are off in this file

File: app/views/messages/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,43 @@
+<h3>Your messages</h3>
```
Comment (harrychenca) [inline app/views/messages/index.html.erb]: Indentations

File: app/views/messages/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+                  <td><%= message.created_at %></td>
+                  
+                  <td><%= link_to "Link to this message", message_path(message.id), :class => "btn btn-xs btn-info" %></td>
+                </tr> 
+            <% end %>
+            </tbody>
+        </table>
+      </div>
+
+    <script>
```
Comment (h8rry) [inline app/views/messages/index.html.erb]: You can remove this

**Comment by vivientam** [comment]
Done!

File: app/controllers/appointments_controller.rb [hunk 5]
```diff
@@ -0,0 +1,30 @@
+class AppointmentsController < ApplicationController
+
+  def new
+    @appointment = Appointment.new
+  end
+
+  def create
+    appointment = appointments.new(appoinment_params)
```
Comment (h8rry) [inline app/controllers/appointments_controller.rb]: should be `appointment = Appointment.new(appoinment_params)`

File: app/controllers/appointments_controller.rb [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    
+    if appointment.save
+      redirect_to appointments_path
+      else
```
Comment (h8rry) [inline app/controllers/appointments_controller.rb]: indentation

File: app/models/message.rb [hunk 7]
```diff
@@ -0,0 +1,7 @@
+class Message < ActiveRecord::Base
+
+  belongs_to :user
+
+  validates :content, presence: true
+  validates :content, length: { minimum: 3 }
```
Comment (h8rry) [inline app/models/message.rb]: The two lines can be reduced into one

`validates :content, presence: true, length: { minimum: 3 }`

File: app/views/layouts/_header.html.erb (hunk 2 above)
Comment (h8rry) [inline app/views/layouts/_header.html.erb:25]: Indentation is still off on line 2

File: app/views/messages/index.html.erb (hunk 3 above)
Comment (harrychenca) [inline app/views/messages/index.html.erb]: Still off

File: app/views/messages/new.html.erb [hunk 8]
```diff
@@ -0,0 +1,14 @@
+<h1>CONTACT US</h1>
+
+<P>Please fill out the fields below to send us a message. For quick questions, or to get the latest news, visit @ colourfulblind on Twitter.</P>
+
+<%= form_for @message do |f| %>
+  <div>
```
Comment (h8rry) [inline app/views/messages/new.html.erb]: Do you need this `<div>`?

**Comment by vivientam** [comment]
Done!

File: app/views/messages/index.html.erb [hunk 9]
```diff
@@ -0,0 +1,39 @@
+<h3>Your messages</h3>
+
+<% if @messages.empty? %>
+
+    <p class="text-general text-reading">
+     You don't have any messages yet...
+    </p>
+
+  <% else %>
```
Comment (h8rry) [inline app/views/messages/index.html.erb]: Indentation

**Pull request merged**